src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Assignment.java

```diff
@@ -43,48 +43,49 @@
 import java.net.URI;
 import java.util.Date;
 import java.util.List;
+import java.util.Optional;
 import java.util.stream.Collectors;
 
 /**
  * Created by jgmeligmeyling on 04/03/15.
  */
 @Data
 @Entity
-@Table(name= "assignments")
-@ToString(exclude = {"summary", "tasks"})
+@Table(name = "assignments")
+@ToString(exclude = {"summary", "tasks", "assignedTAS"})
 @IdClass(Assignment.AssignmentId.class)
-@EqualsAndHashCode(of = {"courseEdition", "assignmentId" })
+@EqualsAndHashCode(of = {"courseEdition", "assignmentId"})
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Assignment implements Comparable<Assignment>, Base {
 
-	public static final String ASSIGNMENTS_PATH_BASE = "assignments/";
+    public static final String ASSIGNMENTS_PATH_BASE = "assignments/";
 
-	@Data
-	@NoArgsConstructor
-	@AllArgsConstructor
-	public static class AssignmentId implements Serializable {
+    @Data
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class AssignmentId implements Serializable {
 
-		private long courseEdition;
+        private long courseEdition;
 
-		private long assignmentId;
+        private long assignmentId;
 
-	}
+    }
 
-	@Id
-	@JsonBackReference
+    @Id
+    @JsonBackReference
     @ManyToOne(optional = false)
     @JoinColumn(name = "course_edition_id")
     private CourseEdition courseEdition;
 
     @Id
-	@JsonProperty("id")
+    @JsonProperty("id")
     @Column(name = "assignment_id")
-	@GeneratedValue(generator = "seq_group_number", strategy = GenerationType.AUTO)
-	@GenericGenerator(name = "seq_group_number", strategy = "nl.tudelft.ewi.devhub.server.database.entities.identity.FKSegmentedIdentifierGenerator", parameters = {
-		@org.hibernate.annotations.Parameter(name = FKSegmentedIdentifierGenerator.TABLE_PARAM, value = "seq_assignment_id"),
-		@org.hibernate.annotations.Parameter(name = FKSegmentedIdentifierGenerator.CLUSER_COLUMN_PARAM, value = "course_edition_id"),
-		@org.hibernate.annotations.Parameter(name = FKSegmentedIdentifierGenerator.PROPERTY_PARAM, value = "courseEdition")
-	})
+    @GeneratedValue(generator = "seq_group_number", strategy = GenerationType.AUTO)
+    @GenericGenerator(name = "seq_group_number", strategy = "nl.tudelft.ewi.devhub.server.database.entities.identity.FKSegmentedIdentifierGenerator", parameters = {
+            @org.hibernate.annotations.Parameter(name = FKSegmentedIdentifierGenerator.TABLE_PARAM, value = "seq_assignment_id"),
```

Comment:
Yes I reformatted it, and no it wasn't intentional; JW pointed it out in another comment as well. Will avoid in the future!